src/Components/home/achievements/components/Card.js

```diff
@@ -0,0 +1,28 @@
+import React, { useEffect, useState } from 'react'
+import { XyzTransition } from '@animxyz/react'
+
+export const Card = ({
+    img = '',
+    imgBgColor = 'bg-red-500',
+    title = '100k+',
+    subtitle = 'Our Happy Students' }) => {
```

Comment:
consider using object as the props for title, subtitle, image.

imgBgColor please remove and accept the className props instead.